app/models/pet_application.rb

```diff
@@ -0,0 +1,8 @@
+class PetApplication < ApplicationRecord
+  belongs_to :pet
+  belongs_to :application
+
+  def self.get_pets
+    self.all.map {|app| Pet.find(app.pet_id)}
+  end
```

Comment:
I will be revising the use of this method. I'm thinking we can add an adopted? boolean in our pets to avoid having to create and use this method.

Reply:
I'm okay with that. I think a boolean would be the easiest route to keep the pets separate.